### PR TITLE
Ch/refactor

### PR DESCRIPTION
**Pull request by njolipatrick**

### CODE REVIEW

File: dist/APIs/controllers/AUTH/authentication.js [hunk 1]
```diff
@@ -0,0 +1,188 @@
+"use strict";
```
Comment (forestbaba) [inline dist/APIs/controllers/AUTH/authentication.js]: Add dist folder to gitignore...It's very important

File: dist/APIs/controllers/ERRORS/errorHandler.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+"use strict";
```
Comment (forestbaba) [inline dist/APIs/controllers/ERRORS/errorHandler.js]: Here too

File: src/v1/apis/Services/auth/auth.service.ts [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+        }
+    }
+    public async login(res: Response, email: string, password: string) {
+        const usercheck: User = await globalQuery.findOne({
+            model: 'users',
+            table: 'email',
+            value: email,
+        });
+
+        // if (!usercheck.is_verified) {
```
Comment (forestbaba) [inline src/v1/apis/Services/auth/auth.service.ts]: Please remove every dead codes

File: dist/APIs/controllers/AUTH/authentication.js (hunk 1 above)
Comment (njolipatrick) [reply dist/APIs/controllers/AUTH/authentication.js]: I already did but it is not ignoring.

File: src/v1/apis/Services/auth/auth.service.ts (hunk 3 above)
Comment (njolipatrick) [reply src/v1/apis/Services/auth/auth.service.ts]: removed

**Review by forestbaba — APPROVED** [review]

**Pull request merged**